ext/enchant/enchant.stub.php

```diff
@@ -2,94 +2,70 @@
 
 /** @generate-function-entries */
 
-/** @return resource|false */
-function enchant_broker_init() {}
+final class EnchantBroker
+{
+}
 
-/** @param resource $broker */
-function enchant_broker_free($broker): bool {}
+final class EnchantDict
```

Comment:
Could we use `EnchantDictionary` instead? The usage of the `Dict` abbreviation doesn't seem to fit PHP's recent naming strategy.

Reply:
I'm okay either way.

Reply:
I used `EnchantDictionary`